src/util/secrets/secrets.c

```diff
@@ -402,9 +402,9 @@ static int local_check_max_payload_size(struct sss_sec_req *req,
     max_payload_size = req->quota->max_payload_size * 1024; /* kb */
```

Comment:
I think comment is wrong here? It shall read 'bytes'.

Reply:
Done.